sphinx_gallery/tests/test_gen_rst.py

```diff
@@ -66,19 +82,7 @@ def test_codestr2rst():
 
 def test_extract_intro():
     with tempfile.NamedTemporaryFile('w') as f:
-        f.write('\n'.join(['"""'
-                           'Docstring header',
-                           '================',
-                           '',
-                           'This is the description of the example',
-                           'which goes on and on',
-                           '',
-                           '',
-                           'And this is a second paragraph',
-                           '"""',
-                           '',
-                           '# and now comes the module code',
-                           'x, y = 1, 2']))
+        f.write('\n'.join(content))
```

Comment:
Why is this changed? I see you are reusing the content, to make it a global variable you should use capital letters when defining it. Nevertheless I see little point in using it for your next test function